From d51fced8e372151f95d0808969f30f7ce3011544 Mon Sep 17 00:00:00 2001
From: Bertrand Aygon <bertrand.aygon@intel.com>
Date: Thu, 28 Jul 2011 12:14:23 +0200
Subject: [PATCH 3/4] ofono: Network name is empty when operator is not
 available

When the operator is not available and the network name is left to NULL,
the service is built with the hidden flag set.
The operator name will be updated whenever oFono passes it to ConnMan.
---
 plugins/ofono.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/plugins/ofono.c b/plugins/ofono.c
index 5a60882..f30c2c2 100644
--- a/plugins/ofono.c
+++ b/plugins/ofono.c
@@ -857,6 +857,8 @@ static int add_network(struct connman_device *device,
 
 	if (modem->operator)
 		connman_network_set_name(network, modem->operator);
+	else
+		connman_network_set_name(network, "");
 
 	if (modem->has_strength)
 		connman_network_set_strength(network, modem->strength);
-- 
1.7.6

